doc/rst/language/spec/statements.rst

```diff
@@ -898,21 +902,19 @@ still be accessible on instances of the type. For example:
       3
       In A.foo()
 
-Within an ``only`` list, a visible symbol from that module may
-optionally be given a new name using the ``as`` keyword. This new name
-will be usable from the scope of the use in place of the old name unless
-the old name is additionally specified in the ``only`` list. If a use
-which renames a symbol is present at module scope, uses of that module
-will also be able to reference that symbol using the new name instead of
-the old name. Renaming does not affect accesses to that symbol via the
-source module’s or enumerated type’s prefix, nor does it affect uses of
-that module or enumerated type from other contexts. It is an error to
-attempt to rename a symbol that does not exist or is not visible in the
-respective module or enumerated type, or to rename a symbol to a name
-that is already present in the same ``only`` list. It is, however,
-perfectly acceptable to rename a symbol to a name present in the
-respective module or enumerated type which was not specified via that
-``only`` list.
+Within an ``only`` list, a visible symbol from that module may optionally be
+given a new name using the ``as`` keyword. This new name will be usable from the
+scope of the use in place of the old name unless the old name is additionally
+specified in the ``only`` list. If a use which renames a symbol is present at
```

Comment:
Again not something you introduce here but "... name will be usable ... unless the old name is additionally ..." can be a bit confusing. We can maybe separate this sentence. I think the part before "unless" is more important than the other and the second part creates some confusion.